infoschema/tables_test.go

```diff
@@ -86,14 +86,14 @@ type testClusterTableSuite struct {
 
 func (s *testClusterTableSuite) SetUpSuite(c *C) {
 	s.testTableSuite.SetUpSuite(c)
-	s.rpcserver = setUpRPCService(c, "0.0.0.0:10080")
+	s.rpcserver = setUpRPCService(c, "0.0.0.0:10080", s.dom)
```

Comment:
Prefer to make `setUpRPCService` as a method of `testClusterTableSuite`.

Reply:
done.